surefire-providers/common-junit4/src/main/java/org/apache/maven/surefire/common/junit4/JUnit4RunListener.java

```diff
@@ -58,11 +63,16 @@
      *
      * @param reporter the reporter to log testing events to
      */
-    public JUnit4RunListener( RunListener reporter )
+    public JUnit4RunListener( TestReportListener reporter )
     {
         this.reporter = reporter;
     }
 
+    public final ConsoleLogger getConsoleLogger()
```

Comment:
There cannot be Override because there is no such super type having a method to override.
This class in another module, so it must be public and of course without Overide - no abstract method in super type.